bindings/lua/tests/CMakeLists.txt

```diff
@@ -16,7 +16,10 @@ macro(add_lua_unit_test luascript)
   endif()
 endmacro()
 
-add_lua_unit_test(test_bottle_and_property.lua)
-add_lua_unit_test(test_resource_finder.lua)
-add_lua_unit_test(test_vocab.lua)
-add_lua_unit_test(test_port.lua)
+
+file(GLOB yarp_lua_test_files "${CMAKE_CURRENT_SOURCE_DIR}/*.lua")
```

Comment:
I do not have a strong opinion on this, but typically on YARP we avoid CMake globbing (see https://stackoverflow.com/questions/32411963/why-is-cmake-file-glob-evil). I guess @drdanz has the last word here.

Reply:
I just copied what there is in other tests(harness)
https://github.com/robotology/yarp/blob/d602780457686d2536cd101fb9f11a0b1621ccf3/tests/CMakeLists.txt#L97-L98

The alternative of globbing is to add each test one by one ??

Reply:
> The alternative of globbing is to add each test one by one ??

Exactly. That's the reason why I do not have a strong opinion on this. Listing all the files is convenient for the reasons outline in https://stackoverflow.com/questions/32411963/why-is-cmake-file-glob-evil , but it is sort of duplicating the presence of the files with a given directory in the filesystem.